staging/src/k8s.io/component-base/logs/api/v1/kube_features.go

```diff
@@ -57,16 +58,23 @@ const (
 	LoggingStableOptions featuregate.Feature = "LoggingStableOptions"
 )
 
-func featureGates() map[featuregate.Feature]featuregate.FeatureSpec {
-	return map[featuregate.Feature]featuregate.FeatureSpec{
-		ContextualLogging: {Default: contextualLoggingDefault, PreRelease: featuregate.Beta},
-
-		LoggingAlphaOptions: {Default: false, PreRelease: featuregate.Alpha},
-		LoggingBetaOptions:  {Default: true, PreRelease: featuregate.Beta},
+func featureGates() map[featuregate.Feature]featuregate.VersionedSpecs {
+	return map[featuregate.Feature]featuregate.VersionedSpecs{
+		ContextualLogging: {
+			{Version: version.MustParse("1.24"), Default: contextualLoggingDefault, PreRelease: featuregate.Alpha},
+			{Version: version.MustParse("1.30"), Default: contextualLoggingDefault, PreRelease: featuregate.Beta},
+		},
+		LoggingAlphaOptions: {
+			{Version: version.MustParse("1.24"), Default: false, PreRelease: featuregate.Alpha},
+		},
+		LoggingBetaOptions: {
+			{Version: version.MustParse("1.24"), Default: true, PreRelease: featuregate.Alpha},
```

Comment:
This entry makes no sense. The whole point of `LoggingBetaOptions` was to mark options which are beta, right at the time that this feature gate was introduced. Please remove this line.

Reply:
Updated. Thanks!